### DISCUSSION TITLE
Configurability of "left to spent" time span

### DISCUSSION THREAD

**Discussion by niklas152** [firefly-iii/firefly-iii#9131] [Firefly III] [answered]
Hello!

As of now, the left to spent display (both on the front page and the budgets page) do not include the current day - i.e., when I have say 30 € left for the month and today is the second to last day (i.e., the 30. of July), the display will say "30 € left to spent". 
However, I find this irritating/misleading, as one could then mistakingly think today would be the last day of the month, happily spent the 30 € and be surprised by the next day with nothing left to spent.
I know this is somewhat a user error, but well, I would still like the ability to configure whether to include the current day (new behaviour, displaying only 15 €) or not (current state, 30 €).
Maybe it's already there and I just didn't see the option?

Thanks and kind regards,
Niklas

**Comment by JC5** [comment, marked as answer]
Hey, great observation!

That's not hard to fix, and will be different in the next release of Firefly III. It makes more sense to count the last day as a day as well.

This will not be configurable, but your example will say "15 to left to spend" in the future.

**Reply by niklas152** [reply]
Oh, very nice, thanks a lot! :)
